.github/workflows/build.yml

```diff
@@ -29,12 +29,11 @@ jobs:
 
     - name: Upload coverage to Codecov
       timeout-minutes: 10
-      # GitHub Actions currently only allow secret usage access during push events
-      if: github.event_name == 'push' && matrix.os == 'ubuntu-18.04' && github.repository == 'hyperledger/aries-framework-go'
-      uses: codecov/codecov-action@v1.0.2
+      if: matrix.os == 'ubuntu-18.04' && github.repository == 'hyperledger/aries-framework-go'
+      uses: codecov/codecov-action@v1.0.6
       with:
-        token: ${{secrets.CODECOV_TOKEN}}
         file: ./coverage.txt
+        fail_ci_if_error: true
```

Comment:
not sure that we should actually fail the workflow if codecov fails.

Reply:
We should definitely not be blocked for days while codecov integration is broken.

I believe there is the option to merge the PR even if the workflow fails. The onus is on the reviewer to inspect the reason why the workflow failed and ensure it's due to this bug.

Reply:
Or I remove this flag and you know that codecov failed because the comment isn't posted.

Reply:
A failed workflow raises a higher red flag than a missing codecov comment imo

Reply:
Actually, now that i think about it, i wouldn't want the build status badge to show "failed " just because this thing broke.

Maybe we should just remove this flag as you say

Reply:
Removed.